### DISCUSSION TITLE
account has been blocked

### DISCUSSION THREAD

**Discussion by Josecep** [vercel/community#3638] [Account Issue] [answered]
### Summary

Hello, I'm not sure why your account has been blocked. Perhaps it was because you forked inappropriate projects from git and have been using Vercel for a while without realizing it. I hope the issue can be resolved.

### Steps to Reproduce

Hello, I'm not sure why your account has been blocked. Perhaps it was because you forked inappropriate projects from git and have been using Vercel for a while without realizing it. I hope the issue can be resolved.

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Comment by Josecep** [comment]
怎能累了就说放弃i
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "vercel/community"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年8月16日(星期三) 晚上10:40
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [vercel/community] account has been blocked (Discussion #3638)





 
The original email was ***@***.*** with the account name being josecep or Josecep. It was also associated with several subdomains, but now none of them are functional.
